Backend/starter_project_mavericks/BlogApp/BlogApp.Application/Features/Ratings/CQRS/Handlers/Create_RatingCommandHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿using AutoMapper;
+using BlogApp.Application.Contracts.Persistence;
+using BlogApp.Application.Exceptions;
+using BlogApp.Application.Features._Indices.CQRS.Commands;
+using BlogApp.Application.Features._Indices.DTOs.Validators;
+using BlogApp.Application.Features.Ratings.CQRS.Commands;
+using BlogApp.Application.Features.Ratings.DTOs.Validators;
+using BlogApp.Application.Responses;
+using BlogApp.Domain;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Features.Ratings.CQRS.Handlers;
+public class Create_RatingCommandHandler : IRequestHandler<Create_RatingCommand, BaseResponse<int>>
```

Comment:
It would be better if we remove the "_" in the class name and file name as well. Do this for all the files and classes.